ui/lib/apps/Statement/pages/List/TimeRangeSelector.tsx

```diff
@@ -124,6 +124,44 @@ export function calcValidStatementTimeRange(
   }
 }
 
+function calcCommonTimeRange(
+  minServerDataTime: number,
+  maxServerDataTime: number
+): { enabled: boolean; value: number }[] {
+  if (!maxServerDataTime) {
+    return RECENT_SECONDS.map((s) => ({ enabled: false, value: s }))
+  }
+
+  const calcInterval =
+    maxServerDataTime +
+    getLocalTimeInterval(maxServerDataTime) -
+    minServerDataTime
+
+  return RECENT_SECONDS.map((s) => ({ enabled: s <= calcInterval, value: s }))
```

Comment:
In my opinion, the following code may be enough?

```js
const now = Date.now()
return RECENT_SECONDS.map(s => ({enabled: now-s >= minminServerDataTime, value:s}))
```

Reply:
I am afraid that the time zone of the database is different. IDK the exact scenario for this.

Reply:
Different from the client maybe.

Reply:
I'm afraid that the time setting on the client is wrong. This can be avoided by using time intervals.